tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -196,23 +197,18 @@
                 <ColumnDefinition Width="Auto" />
             </Grid.ColumnDefinitions>
             <TextBlock Grid.Column="0" x:Uid="EnvironmentsHeader" Style="{ThemeResource SubtitleTextBlockStyle}" HorizontalAlignment="Left" x:Name="EnvironmentsHeader"/>
-            <Button 
-                Grid.Column="3" 
+            <views:AddCreateButton
+                Grid.Column="3"
+                Visibility="{x:Bind ViewModel.ShouldNavigateToExtensionsPage, Mode=OneWay, Converter={StaticResource NegatedBoolToVisibilityConverter}}"
                 HorizontalAlignment="Right"
-                Command="{x:Bind ViewModel.CallToActionInvokeButtonCommand}" >
-                <Grid>
-                    <TextBlock 
-                        Visibility="{x:Bind ViewModel.ShouldNavigateToExtensionsPage, Mode=OneWay, Converter={StaticResource NegatedBoolToVisibilityConverter}}"
-                        Grid.Column="1" 
-                        x:Uid="CreateEnvironmentButton" 
-                        Margin="3 0 0 0"  />
-                    <TextBlock 
-                        Visibility="{x:Bind ViewModel.ShouldNavigateToExtensionsPage, Mode=OneWay}"
-                        Grid.Column="1" 
-                        x:Uid="GoToExtensionsLibrary" 
-                        Margin="3 0 0 0"  />
-               </Grid>
-            </Button>
+                x:Uid="CreateEnvironmentButton"
+                Command="{x:Bind ViewModel.CallToActionInvokeButtonCommand}" />
+            <Button
+                Grid.Column="3"
+                Visibility="{x:Bind ViewModel.ShouldNavigateToExtensionsPage, Mode=OneWay}"
+                HorizontalAlignment="Right"
+                x:Uid="GoToExtensionsLibrary"
+                Command="{x:Bind ViewModel.CallToActionInvokeButtonCommand}" />

```

Comment:
Why isn't this a `AddCreationButton` too?

Reply:
Here they're just navigating to another page, I don't think we want a + icon.